docs/source/documentation/case-booking.html.md.erb

```diff
@@ -0,0 +1,32 @@
+---
+title: Case Bookings
+---
+
+# Case Bookings
+Cases are created at the contact stage of a users journey. When the user selects `Submit details`, the full payload is sent to the 
+CLA backend endpoint which via `post_case`. This sends the full payload to backend with all required information including information 
```

Comment:
can we be consistent with naming `CLA Backend`, ie capitalise the `B` as you have done elsewhere. If we are just writing `backend` it should just be lower case.